Mh/hunter dog/superbranch

bleehu opened the pull request:
Voila! New hunter dog!

bleehu committed:
added some rules about attack dogs

bleehu committed:
added tricks

bleehu committed:
Merge pull request #481 from bleehu/MH/Hunter-Dog/Add-tricks-doc

added tricks

bleehu committed:
Revert "added tricks"

bleehu committed:
Merge pull request #485 from bleehu/revert-481-MH/Hunter-Dog/Add-tricks-doc

Revert "added tricks"

bleehu committed:
Merge pull request #482 from bleehu/MH/add-dog-rules

added some rules about attack dogs

bleehu committed:
Added Hunter's Dog document,
As well as basic rules for dog and reference in classes doc. Added warning about projection
onto the dog. Attempting to keep dog mechanics very simple. Though I don't think I've
achieved my goal of making the hunter class feel like predator.

Changes to be committed:
	modified:   02_Classes.txt
	new file:   Class-Specific Documentation/Hunters_Dog.txt

bleehu committed:
Merge branch 'dev' into MH/Hunter-Dog/Superbranch

bleehu committed:
added dog food item

bleehu committed:
added dog toy item

bleehu committed:
added dog collar/leash

bleehu committed:
changed description of hunter class; no longer an emphasis on nanite usage

bleehu committed:
Removed old hunter nanite class feats. Added Tracker, Spotter and Pack Hunter feats
	modified:   CharacterCreation/02_Classes.txt

bleehu committed:
Merge pull request #583 from bleehu/MH/Hunter-Dog/DogStats

Mh/hunter dog/dog stats

bleehu committed:
Merge pull request #584 from bleehu/MH/Hunter-Dog/Dogfood

added dog food item

bleehu committed:
Merge pull request #587 from bleehu/MH/Hunter-Dog/description

changed description of hunter class

bleehu committed:
Changed collar to harness

bleehu committed:
removed 'crit' from hunter class feats

bleehu committed:
Merge pull request #588 from bleehu/MH/Hunter-Dog/ClassFeats

Mh/hunter dog/class feats

bleehu committed:
removed line about getting attention; trained attack dogs don't need
toys for getting attention.

bleehu committed:
Merge pull request #586 from bleehu/MH/Hunter-Dog/Collar

added dog collar/leash

bleehu committed:
Merge pull request #585 from bleehu/MH/Hunter-Dog/chewtoy

added dog toy item

bleehu committed:
removed sniper rifle from hunter class

bleehu commented:
Please refer to design constraints to see how it measures up:
https://github.com/bleehu/CX_Design_Guide/blob/master/characters/how_to_classes.md#hunter

bleehu committed:
added new class feat to hunters: Matador

bleehu committed:
changed name of hunter class feat from Matatdor to Bushwhacker

bleehu committed:
changed had to wielding in bushwacker feat for hunter

bleehu committed:
Merge pull request #617 from bleehu/MH/Hunter-Dog/Matador

Mh/hunter dog/matador

rosebrallier commented on CharacterCreation/02_Classes.txt:
```diff
@@ -427,34 +428,97 @@ Skill Proficiencies:
+the "Items/07_Items.txt" document) or that are within 5m of you. Party Members
+(including yourself) that shoot at an indicated target get a +1 to accuracy 
+rolls to hit that target. A target remains indicated until they move, or for
+two Rounds, whichever comes first. You also get 6 skill points in Investigate
```
*Investigation

rosebrallier commented on CharacterCreation/02_Classes.txt:
```diff
@@ -427,34 +428,97 @@ Skill Proficiencies:
+when you take this Class Feat. 
+
+Spotter:
+	Requires that your character be level 10 and that you have the "Tracker"
```
Subject consistency: "that your character be level 10 and have the..."
(singular subject cannot be both "your character" and "you")

rosebrallier commented on CharacterCreation/02_Classes.txt:
```diff
@@ -427,34 +428,97 @@ Skill Proficiencies:
+
+Spotter:
+	Requires that your character be level 10 and that you have the "Tracker"
+class feat. The first time each turn that you track an enemy that you already
```
Imo "have already" reads more naturally

rosebrallier commented on CharacterCreation/02_Classes.txt:
```diff
@@ -427,34 +428,97 @@ Skill Proficiencies:
+	Requires that your character be level 10 and that you have the "Tracker"
+class feat. The first time each turn that you track an enemy that you already
+tracked this encounter, you may do so as a Free Action instead of an Action.
+	For example, let's say a Hunter was fighting an enemy robot and an enemy 
```
"was" --> "is" (b/c the rest of the passage is in present tense)

rosebrallier commented on CharacterCreation/02_Classes.txt:
```diff
@@ -427,34 +428,97 @@ Skill Proficiencies:
+|   R2  |    10%     |             25%             |
++-------+------------+-----------------------------+
+|   R3  |    20%     |            100%             |
++-------+------------+-----------------------------+
```
Do we want these to be written as percentages, or flat bonuses to Critical Range? Right now, this mechanic implies that teammates multiply their Crit range by 1.**, which is a potentially bothersome amount of math-on-the-fly.

rosebrallier replied:
@bleehu

bleehu replied:
How's that update?

rosebrallier commented on CharacterCreation/02_Classes.txt:
```diff
@@ -427,34 +428,97 @@ Skill Proficiencies:
+Pack Hunter (II):
+	Requires that your character be level 10 and that you have Pack Hunter (I).
+Your bonus for shooting at an enemy suppressed by your party member now includes
+a +5% chance to critically hit. The bonus for melee for both you and your 
```
Same comment as above re:percentages. With this one, since there's only one person making one calculation it's not bad on the math front, but we should use a consistent format for crit range bonuses.

Reveraine replied:
I agree with Rose. Especially with our dice rolls being less granular.

rosebrallier commented on CharacterCreation/02_Classes.txt:
```diff
@@ -427,34 +428,97 @@ Skill Proficiencies:
+
+Bushwhacker:
+	You've had enough toothy critters jump out of the brush and try to bite your
+head off to be pretty good with shooting things at close range. You get 
```
I usually use "gain advantage." It's personal preference, but we should be consistent.

rosebrallier commented on CharacterCreation/02_Classes.txt:
```diff
@@ -427,34 +428,97 @@ Skill Proficiencies:
+Bushwhacker:
+	You've had enough toothy critters jump out of the brush and try to bite your
+head off to be pretty good with shooting things at close range. You get 
+advantage on shooting targets that are weilding a melee weapon with your primary 
```
"wielding"

rosebrallier commented on CharacterCreation/02_Classes.txt:
```diff
@@ -427,34 +428,97 @@ Skill Proficiencies:
+	You've had enough toothy critters jump out of the brush and try to bite your
+head off to be pretty good with shooting things at close range. You get 
+advantage on shooting targets that are weilding a melee weapon with your primary 
+weapon within your weapon's R1 range bracket. You also get advantage on 
```
This sentence has a lot of (3) subordinate clauses; it can be unwieldy to read and parse a sentence with a lot of consecutive sub. clauses. 
Additional question: does that target have to be attacking with a melee weapon, or can it include CQB weapons and/or melee creature attacks (like bite- or claw-attacks). I can imagine this getting house-ruled pretty quickly to explicitly include creatures and unarmed assailants.

Turtlelord26 replied:
The rules formally consider Unarmed Strike to be a melee weapon, which I would think implies natural weapons like teeth by association. That said, it wouldn't cost much to be more explicit and it might help somebody dodge a rules lawyering assassin player.

rosebrallier commented on CharacterCreation/Class-Specific Documentation/Hunters_Dog.txt:
```diff
@@ -0,0 +1,57 @@
+    The Dog cannot use weapons, armor nor non-food items. 
+
+    The Dog is smaller than most combatants and moves very quickly. If the Dog 
+moved at least 8m on their last turn, they are considered to be in partial cover
```
"has moved"

rosebrallier commented on CharacterCreation/Class-Specific Documentation/Hunters_Dog.txt:
```diff
@@ -0,0 +1,57 @@
+Death Saving Throws: 3
+Nanites:    0
+
+    The Dog has a +(level x 4) bonus to investigate and awareness checks that 
```
"Investigation checks and Awareness saves"

rosebrallier commented on CharacterCreation/Class-Specific Documentation/Hunters_Dog.txt:
```diff
@@ -0,0 +1,57 @@
+Nanites:    0
+
+    The Dog has a +(level x 4) bonus to investigate and awareness checks that 
+they can use their sense of smell to help with. 
```
sen. structure pref: "that their sense of smell can help with"

rosebrallier commented on CharacterCreation/Class-Specific Documentation/Hunters_Dog.txt:
```diff
@@ -0,0 +1,57 @@
+
+    The Dog can use tricks like "lay down," "shake," "fetch," or "chase your 
+tail" to charm or distract characters in the area. When they do, the target must
+succeed on a Will save opposed by the dog's Charisma (+12) performance roll.
```
Performance is a skill, so we may want to either say "the dog's Charisma (+12) roll" or use standard skill notation- "the dog's Performance (Cha) roll".

rosebrallier commented on CharacterCreation/Class-Specific Documentation/Hunters_Dog.txt:
```diff
@@ -0,0 +1,57 @@
+    The Dog's primary weapon is their bite. Their melee roll is 
+6 + (level/4) + roll. On a successful hit, the target is grappled with the Dog
+in control. Every time the Dog makes a melee attack while it is in control of a
+grapple, it deals 60 damage. 
```
We may want to clarify whether the initial grapple attack deals damage.

rosebrallier commented on Items/07_Items.txt:
```diff
@@ -2,6 +2,10 @@
+Dog Toy ($20):
+    This brightly colored rubber ball is available in both the squeaking and 
+not-squeaking varieties. Very Bouncy and great for tricks like fetch.
+
```
XD

rosebrallier commented on CharacterCreation/Class-Specific Documentation/Hunters_Dog.txt:
```diff
@@ -0,0 +1,57 @@
+
+== Projecting ==
+
+In testing, we have found that players who love animals (sometimes beloved 
```
While I agree with concerns about projecting, have we actually had this come up in testing? (aka Have we done enough testing for this to actually come up?) We may want to just delete "In testing, we have found that".

bleehu replied:
This particular testing wasn't run in compound x, but was found in my earlier time playing similar games as a child. I think it's valuable to show that this is a thing that has happened before, and that if a player is projecting that they're not alone in experiencing this kind of empathy.

rosebrallier commented on Items/07_Items.txt:
```diff
@@ -43,6 +47,12 @@ from multiple angles to capture the whole object. Up to 20m^3 of space.
 
+Dog Harness ($45):
+    This harness comes with up to 18 characters of monogram and a micro
+chip that helps you find your dog if it runs away. The collar itself is 
```
"harness itself"
Most of the time, collars and harnesses have ID tags, while the microchip is implanted directly into the animal (so they can be identified/found in the absence of any equipment).

bleehu replied:
This doesn't preclude putting a chip in the harness as well.

rosebrallier reviewed (changes requested):
A bunch of grammar notes, plus some clarity, mechanics and animal husbandry notes.
Overall, I'm SUPER EXCITED TO PLAY THIS OMG.

bleehu committed:
Fixed spelling and grammar errors from review
	modified:   CharacterCreation/02_Classes.txt
	modified:   CharacterCreation/Class-Specific Documentation/Hunters_Dog.txt
	modified:   Items/07_Items.txt

bleehu committed:
clarified spotter class feat for hunters

Reveraine commented on CharacterCreation/02_Classes.txt:
```diff
@@ -388,9 +388,15 @@ Skill Proficiencies:
+	Unlike many of your compatriots who prefer the cold comforts of technology,
+you have an affinity for the frontier. The great outdoors. Be they the 
+wind-swept plains of frontier farms, the thick jungles of Tundaar or the swamps
+of Trioch you love it all. You are much more likely to know which plants won't 
```
I think there should be some punctuation here: of Trioch; you love it all.

Turtlelord26 replied:
Semicolon is a bit of overkill, but a comma or dash would add to the text.

Reveraine commented on CharacterCreation/02_Classes.txt:
```diff
@@ -427,34 +428,100 @@ Skill Proficiencies:
+you found the dog along your last mission, or maybe a dog was issued to you when
+you returned to home base (at the GM's choice). Every time you level up after
+you have your dog, your dog levels up as well as defined in the 
+"Class-Specific Documentation/Hunter's_Dog.txt" document. For simplicity, Dogs 
```
I think this sentence can be condensed a bit. 
For instance "Once you have a dog, it will level up with you, following the rules found in....document.

Reveraine commented on CharacterCreation/02_Classes.txt:
```diff
@@ -427,34 +428,100 @@ Skill Proficiencies:
+you have your dog, your dog levels up as well as defined in the 
+"Class-Specific Documentation/Hunter's_Dog.txt" document. For simplicity, Dogs 
+start at level 3 so that they are always the same level as their owners. If your
+dog is killed in action, you do not get to roll another one. If you and your dog
```
I think this makes it sound like you can never have another dog. I think something better might be "you are not automatically issued a replacement" which feels more like you have to work/find a new dog. 
On the other hand, I really like the reminder that your dog is not expendable, and there are consequences to losing it.

rosebrallier commented on CharacterCreation/02_Classes.txt:
```diff
@@ -480,23 +480,25 @@ Hunter still has 4 Actions left on his turn. Continuing his third turn, the
+humidity, wind speed and direction, and the coriollis effect making the bullet 
+swerve. The bonus to critically hit is listed below. Note that if the chance to 
+critically hit exceeds the chance to hit, it does not increase the chance to 
+hit; it only garuntees that a hit will be a critical hit. If the increase in
```
nb: guarantees

rosebrallier reviewed (approved):
Shipit!!!

Reveraine commented on CharacterCreation/02_Classes.txt:
```diff
@@ -427,34 +428,100 @@ Skill Proficiencies:
+	As an Action, you can study the position and disposition of a target that
+you can see, and indicate it to your Party Members that can communicate with 
+you, either by being on comms with you (see the headset and similar items in
+the "Items/07_Items.txt" document) or that are within 5m of you. Party Members
```
I assume that the within 5m is to indicate secrecy. It isn't directly stated that this is being done quietly, and so the 5m seems like a hard limit on the hearing range of your team. I don't think that was the intention.
Also, could this be indicated at a longer range quietly by using hand-signals?

bleehu replied:
I have decided not to change this one. Describing quickly and accurately where an enemy is and what their stance is requires both words and hand signals at the same time. Doing it over gunfire requires that your ally be close. 
Mechanically, this also helps lend itself to the design goal of further cementing the Hunter class as a pack hunter, who needs to be close to at least one ally to be in their pack.

Reveraine commented on CharacterCreation/02_Classes.txt:
```diff
@@ -427,34 +428,100 @@ Skill Proficiencies:
+listens to you and does his best to make his or her owner happy, but is not 
+controlled by the player. Instead, the owner character commands the dog, and the
+GM controls how the dog responds. 
+
```
I like this functionality! I think it is much more true to animal handling than letting the player control the dog directly.

Reveraine commented on CharacterCreation/02_Classes.txt:
```diff
@@ -427,34 +428,100 @@ Skill Proficiencies:
+the target and describing shooting conditions such as the bullet drop due to 
+humidity, wind speed and direction, and the coriollis effect making the bullet 
+swerve. The bonus to critically hit is listed below. Note that if the chance to 
+critically hit exceeds the chance to hit, it does not increase the chance to 
```
The language about critical bonuses is confusing. 
I think an example may help. The math does not seem to be laid out, but other parts of critical range bonuses are here. I think there should be a reference to the section of the rules that involve critical range increases, so this can be condensed.

bleehu replied:
There does not currently exist good language in our rules for me to hook into. This is as good as it's gonna get.

Reveraine commented on CharacterCreation/02_Classes.txt:
```diff
@@ -427,34 +428,100 @@ Skill Proficiencies:
+advantage on shooting targets with your primary weapon within your weapon's R1 
+range bracket. This target must also either be wielding a melee weapon or be a
+non-sentient enemy that has melee attack abilities. You also get advantage on 
+Perception rolls to spot cloaked or sneaking characters.
```
I like this ability a lot! This will also make hunters good at clearing rooms!

Reveraine commented on CharacterCreation/Class-Specific Documentation/Hunters_Dog.txt:
```diff
@@ -0,0 +1,64 @@
+
+    The Dog is smaller than most combatants and moves very quickly. If the Dog 
+has moved at least 8m on their last turn, they are considered to be in partial 
+cover from all angles. 
```
Yay! Hard to hit rules!

Reveraine commented on CharacterCreation/Class-Specific Documentation/Hunters_Dog.txt:
```diff
@@ -0,0 +1,64 @@
+	Every time the Dog makes a melee attack while it is in control of a
+grapple, it deals 60 damage. Dog attacks usually involve the dog either making
+subsequent bites, and/or shaking their head viciously to shred what they're 
+currently biting on to ribbons.
```
Grammar suggestion: Dogs have two primary methods of attacking: biting a new section of the target, or shaking their head viciously to tear apart a portion they are currently biting.

Reveraine reviewed (changes requested):
Overall, I really love the work put into this! I'm very excited to give this a test run and see it make its way into play!
There are some minor grammar changes, but I think the ideas and execution as a whole are fantastic.

bleehu committed:
Merge branch 'dev' into MH/Hunter-Dog/Superbranch

bleehu committed:
grammar check for new hunter
	modified:   CharacterCreation/02_Classes.txt

bleehu commented:
How are those changes?
